jwst/saturation/tests/test_saturation.py

```diff
@@ -213,11 +213,11 @@ def test_nans_in_mask(setup_nrc_cube):
     correct_for_NaN(satmap.data, satmap.dq)
```

Comment:
I don't think this needs to be in the test, as this gets done in `do_correction()`.

Reply:
Agree. It's been removed.